routes/index.js

```diff
@@ -1,66 +1,63 @@
-var express = require('express');
-var router = express.Router();
-var http = require('http');
-var request = require('request');
+const express = require('express');
+const router = express.Router();
+const http = require('http');
+const request = require('request');
 const getip = require('client-ip');
 const fs = require('fs');
 const path = require('path');
 const jsonParser=require('json-parser');
-
-
-/* GET home page. */
-// way 1 using request
-var options = {
-    method: 'get',
-    json: true,
-    url: "https://www.numbeo.com/api/city_prices?api_key=<API_Key>&query=Belgrade"
+const fetch = require('node-fetch');
+
+
+const getCity = function (userIP){
+    return new Promise(function(resolve, reject) {
+        fetch('https://freegeoip.net/json/100.105.14.101')
+            .then(function(res) {
+                return res.json();
+            }).then(function(localeJSON) {
+            return localeJSON.city;
+        }).then(function(city){
+            city=(city == "" ? "Belgrade" : city);
+            fetch("https://www.numbeo.com/api/city_prices?api_key=<API_Key>&query="+city)
+                .then(function(res){
+                    return res.json();
+                }).then(function(cityStatsJSON) {
+                resolve(cityStatsJSON);
```

Comment:
then here simply return the cityStatsJSON.

this way, the result of calling getCity will be a promise containing the cityStatsJSON value.